src/app/modules/mobile-menu/mobile-menu.component.ts

```diff
@@ -26,7 +25,7 @@ export class MobileMenuComponent implements OnInit {
 
   getUser() {
     this.userService.getUser().subscribe((user) => {
-      if (user && user.role) {
+      if (user.roles?.length > 0) {
         this.signedIn = true;
       } else {
         this.signedIn = false;
```

Comment:
Rewrite this if/else block with in one line?
``` this.signedIn = user.roles?.length > 0 ```

Reply:
Ok, I've made that change.